tests/test_codeblock.py

```diff
@@ -880,6 +919,42 @@ def test_append_with_markdown_header_and_bare_backticks():
     assert "More content that gets lost" in content
 
 
+def test_append_with_markdown_header_and_bare_backticks_streaming():
+    """
+    Streaming mode variant of test_append_with_markdown_header_and_bare_backticks.
+
+    In streaming mode, markdown headers (## Subtitle) followed by bare backticks
+    can cause the parser to incorrectly detect block closure, cutting off content.
+
+    This test verifies the parser correctly handles this pattern during streaming,
+    waiting for blank line confirmation before treating the closing fence as final.
+    """
+    fence = "```"
+    markdown = f"""{fence}append journal.md
+# Journal Entry
+
+Some initial content here.
+
+## Subtitle
+{fence}
+This content after the bare backticks should be included but isn't.
+
+## Another Section
+More content that gets lost.
+{fence}
+
+"""  # Blank line confirms completion in streaming mode
+
+    blocks = list(_extract_codeblocks(markdown, streaming=True))
+    assert len(blocks) == 1, "Should extract complete block in streaming mode"
```

Comment:
Same as the previous review comment, this isn't a complete codeblock so shouldn't get extracted (yet).